### PR TITLE
Remove Isomorphic Fetch from Axios example

### PR DESCRIPTION
**Pull request by jmichaliga**
Cuts down on confusion and a dependency.

### CODE REVIEW

File: examples/axios/pages/api/data.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const projects = [
 export default (req, res) => {
   if (req.query.id) {
     // a slow endpoint for getting repo data
-    fetch(`https://api.github.com/repos/${req.query.id}`)
+    axios(`https://api.github.com/repos/${req.query.id}`)
       .then(resp => resp.json())
```
Comment (shuding) [inline examples/axios/pages/api/data.js:11]: Does axios support this?

Comment (jmichaliga) [reply examples/axios/pages/api/data.js:11]: Yes, for a standard 'GET' request. I'll update the Typescript example as well.

Comment (shuding) [reply examples/axios/pages/api/data.js:11]: It seems `axios` doesn't support `.json()`:

image

https://runkit.com/shuding/5f3e51784380ec001acd0501

Comment (jmichaliga) [reply examples/axios/pages/api/data.js:11]: You're correct. The response requires the `.data` object for axios. I've updated the other example in a separate PR and can close this one.

**Pull request closed without merging**